File: src/main/java/ffblockly/meta/FFBlocklyFieldValue.java
package ffblockly.meta;

import java.util.List;

/**
 * Created by Jnorthrup on 9/13/2016.
 */
public interface FFBlocklyFieldValue {
    String getValue();

    FFBlocklyInputValueBuilder.FFBFieldType getType();


    String getText();

    String getMin();

    String getMax();

    Boolean getChecked();

    List<List<String>>getOptions();

    String getName();

    String getTooltip();
    String getCheck();

}

File: src/main/java/ffblockly/meta/FFBlocklyFieldValue.kt
package ffblockly.meta

/**
 * Created by Jnorthrup on 9/13/2016.
 */
interface FFBlocklyFieldValue {
    val value: String?
    val type: FFBlocklyInputValueBuilder.FFBFieldType?
    val text: String?
    val min: String?
    val max: String?
    val checked: Boolean?
    val options: List<List<String?>?>?
    val name: String?
    val tooltip: String?
    val check: String?
}